src/CssInliner.php

```diff
@@ -1012,39 +1011,40 @@ private function removeUnmatchablePseudoComponents(string $selector): string
 
         // The regex allows nested brackets via `(?2)`.
         // A space is temporarily prepended because the callback can't determine if the match was at the very start.
-        $selectorWithoutNots = \ltrim((new Preg())->throwExceptions($this->debug)->replaceCallback(
-            '/([\\s>+~]?+):not(\\([^()]*+(?:(?2)[^()]*+)*+\\))/i',
-            /** @param array<array-key, string> $matches */
-            function (array $matches): string {
-                return $this->replaceUnmatchableNotComponent($matches);
-            },
-            ' ' . $selector
-        ));
+        $pattern = '/([\\s>+~]?+):not(\\([^()]*+(?:(?2)[^()]*+)*+\\))/i';
+        /** @param array<array-key, string> $matches */
+        $callback = function (array $matches): string {
+            return $this->replaceUnmatchableNotComponent($matches);
+        };
+        $selectorWithoutNots = (new Preg())->throwExceptions($this->debug)
+            ->replaceCallback($pattern, $callback, ' ' . $selector);
+        $trimmedSelectorWithoutNots = \ltrim($selectorWithoutNots);
```

Comment:
Would it be better to name the new intermediate variable `$untrimmedSelectorWithoutNots`, keeping the same name for the final result variable, so that the code which uses it doesn't need to be changed?